### ISSUE TITLE
Render Flow object types as open

### ISSUE THREAD

**Issue by bbqbaron** [rrdelaney/ReasonablyTyped#26]
While researching #19, I wonder if there's an upstream change we need to make. We render Flow object types as closed Reason objects, but I think they're supposed to be open, aren't they?

Right now, test `object-type`:
```
declare type subscribeOptions = {
  start: number,
  stop: number
}
```
becomes

```
type subscribeOptions = Js.t {. start : float, stop : float};
```

However, Flow permits extra fields in object types that aren't [exact](https://flow.org/en/docs/types/objects/#toc-exact-object-types). For example, `function foo(x: SubscribeOptions)` should accept objects with more than just `start` and `stop` on them.

Should we produce instead `Js.t {.. start: float, stop: float }` unless the type is declared with `{| |}`?

**Comment by rrdelaney** [comment]
This seems correct to me! We should definitely try to output as close to Flow bindings as possible, and match the default of open types!

**Comment by rrdelaney** [comment]
Created #27 for supporting closed object types after we move to outputting open ones

**Comment by rrdelaney** [comment]
Closed by #29. Thanks @bbqbaron!
